esp-wifi/MIGRATING-0.10.md

```diff
@@ -17,6 +17,11 @@ The following error enum variants have been removed from `InitializationError`:
 - `TimerUnavailable`
 - `RadioClockUnavailable`
 
+```diff
+- let init: EspWifiInitialization = esp_wifi::init(EspWifiInitFor::Wifi, timg0.timer0, rng.clone(), peripherals.RADIO_CLK).unwrap();
++ let init: EspWifiController = esp_wifi::init(timg0.timer0, rng.clone(), peripherals.RADIO_CLK).unwrap();
```

Comment:
And now there is no way to move the returned value into whatever would need them, but must be borrowed? :/